scripts/mongo-create-user.sh

```
@@ -0,0 +1,14 @@
+mongosh --port 10021 <<EOF
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)